examples/daisyworld.jl

```diff
@@ -318,11 +315,12 @@ Random.seed!(165) # hide
 model = daisyworld(; solar_luminosity = 1.0)
 
 agent_df, model_df = run!(model, agent_step!, model_step!, 1000; adata)
-
-p = plot(agent_df[!, :step], agent_df[!, :count_black_daisies], label = "black")
-plot!(p, agent_df[!, :step], agent_df[!, :count_white_daisies], label = "white")
-plot!(p; xlabel = "tick", ylabel = "daisy count")
-
+figure = Figure()
+ax = figure[1, 1] = Axis(figure, xlabel="tick", ylabel="daisy count")
+blackl = lines!(ax, agent_df[!, :step], agent_df[!, :count_black_daisies], color=:red)
+whitel = lines!(ax, agent_df[!, :step], agent_df[!, :count_white_daisies], color=:blue)
+figure[1, 2] = Legend(figure, [blackl, whitel], ["black", "white"], textsize=12)
+Makie.save("figure1.png", figure) # hide
```

Comment:
Remove the save line, Documenter.jl will display the plot in-line so long as the last line in this block is `figure`.